operator/main.go

```diff
@@ -126,6 +126,9 @@ func init() {
 	flags.Int(option.AWSClientBurst, 4, "Burst value allowed for the AWS client used by the AWS ENI IPAM")
 	flags.Float64(option.AWSClientQPSLimit, 20.0, "Queries per second limit for the AWS client used by the AWS ENI IPAM")
 
+	flags.Float64(option.K8sClientQPSLimit, 0, "Queries per second limit for the K8s client")
```

Comment:
Instead of hardcoding zero, add to `pkg/defaults` a value which encodes what the default value is, e.g., `K8sClientQPSLimit` and `K8sClientBurst`.